pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -7044,6 +7044,92 @@ var _ = Describe("VirtualMachine", func() {
 				Entry("should not appear if VM rollout strategy is set to LiveUpdate",
 					&liveUpdate, Not(restartRequiredMatcher(k8sv1.ConditionTrue))),
 			)
+
+			Context("for hotplugged volumes", func() {
+				addVolume := func() {
+					hpSource := v1.HotplugVolumeSource{
+						DataVolume: &v1.DataVolumeSource{
+							Name:         "hotplug-dv",
+							Hotpluggable: true,
+						},
+					}
+					disk := v1.Disk{
+						Name: "hotplug",
+						DiskDevice: v1.DiskDevice{
+							Disk: &v1.DiskTarget{Bus: v1.DiskBusSCSI},
+						}}
+					vm.Status.VolumeRequests = append(vm.Status.VolumeRequests, v1.VirtualMachineVolumeRequest{
+						AddVolumeOptions: &v1.AddVolumeOptions{
+							Name:         "hotplug",
+							VolumeSource: &hpSource,
+							Disk:         &disk,
+						},
+					})
+					vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.TODO(), vm, metav1.CreateOptions{})
+					Expect(err).To(Succeed())
+					addVirtualMachine(vm)
+
+				}
+				It("shouldn't appear with the stage rollout strateg and when a volume is added with a hotplug volume request", func() {
+					kv.Spec.Configuration.VMRolloutStrategy = pointer.P(stage)
+					testutils.UpdateFakeKubeVirtClusterConfig(kvStore, kv)
```

Comment:
Put this into `BeforeEach` of the context and also add `with the stage rollout strategy` to the context?